src/fillTank.test.js

```diff
@@ -1,11 +1,115 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared', () => {
+    expect(typeof (fillTank)).toBe('function');
+  });
+
+  it('If the amount is not given, then full tank is ordered.', () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 1);
+
+    expect(customer.money).toBe(2968);
+  });
+
+  it('If the amount is greater than the tank can accommodate,'
+    + ' pour only what will fit.', () => {
```

Comment:
This test case correctly checks if the function handles an amount greater than the tank's capacity. Ensure that the `fillTank` function logic aligns with this test to pour only what will fit.